providers/fetcher/githubFetcher.js

```diff
@@ -25,7 +25,7 @@ class GitHubFetcher {
 
   _fetchFromGitHub(request, checkEtag) {
     const self = this;
-    const etagPromise = checkEtag ? this.store.etag(request.type, request.url) : Q(null);
+    const etagPromise = checkEtag ? this.store.etag(request.type, request.context.cacheKey || request.url) : Q(null);
```

Comment:
revert this change and the one later in this method.  Should never have this case.